test/parallel/test-async-wrap-check-providers.js

```diff
@@ -75,11 +75,11 @@ process.on('SIGINT', () => process.exit());
 
 // Run from closed net server above.
 function checkTLS() {
-  let options = {
+  const options = {
     key: fs.readFileSync(common.fixturesDir + '/keys/ec-key.pem'),
     cert: fs.readFileSync(common.fixturesDir + '/keys/ec-cert.pem')
   };
-  let server = tls.createServer(options, noop).listen(common.PORT, function() {
+  const server = tls.createServer(options).listen(common.PORT, function() {
```

Comment:
@silverwind I don't think it will affect the behaviour as `noop` has no executable statements, so in this particular case, it is as good as not attaching.